src/link.stories.ts

```diff
@@ -51,9 +51,9 @@ const meta: Meta = {
     /* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
     return html`
       <glide-core-link
+        label=${arguments_.label || nothing}
```

Comment:
I feel like the required attribute should be rendered first in the code example.

Reply:
I dig it